daemon/health.go

```diff
@@ -325,3 +321,13 @@ func min(x, y int) int {
 	}
 	return y
 }
+
+func getShell(config containertypes.Config) []string {
```

Comment:
Hum I like the idea of `Shell()` method on `Config` :angel:
Right now `Shell` is only used while building to set the right `Entrypoint` — it's not used during *runtime*. If `Shell` is empty, it means it has not be set and thus we should use the default behavior (i.e. either `Container.Entypoint` or the current values).